assets/ios/swift/CompoundCoreColorTokens.swift

```diff
@@ -0,0 +1,320 @@
+    public static let orange900 = Color("orange900", bundle: Bundle.module) 
+    public static let orange800 = Color("orange800", bundle: Bundle.module) 
+    public static let orange700 = Color("orange700", bundle: Bundle.module) 
+    public static let orange600 = Color("orange600", bundle: Bundle.module) 
+    public static let orange500 = Color("orange500", bundle: Bundle.module) 
+    public static let orange400 = Color("orange400", bundle: Bundle.module) 
+    public static let orange300 = Color("orange300", bundle: Bundle.module) 
+    public static let orange200 = Color("orange200", bundle: Bundle.module) 
+    public static let orange100 = Color("orange100", bundle: Bundle.module) 
+    public static let red1400 = Color("red1400", bundle: Bundle.module) 
+    public static let red1300 = Color("red1300", bundle: Bundle.module) 
+    public static let red1200 = Color("red1200", bundle: Bundle.module) 
+    public static let red1100 = Color("red1100", bundle: Bundle.module) 
+    public static let red1000 = Color("red1000", bundle: Bundle.module) 
+    public static let red900 = Color("red900", bundle: Bundle.module) 
+    public static let red800 = Color("red800", bundle: Bundle.module) 
+    public static let red700 = Color("red700", bundle: Bundle.module) 
+    public static let red600 = Color("red600", bundle: Bundle.module) 
+    public static let red500 = Color("red500", bundle: Bundle.module) 
+    public static let red400 = Color("red400", bundle: Bundle.module) 
+    public static let red300 = Color("red300", bundle: Bundle.module) 
+    public static let red200 = Color("red200", bundle: Bundle.module) 
+    public static let red100 = Color("red100", bundle: Bundle.module) 
+    public static let gray1400 = Color("gray1400", bundle: Bundle.module) 
+    public static let gray1300 = Color("gray1300", bundle: Bundle.module) 
+    public static let gray1200 = Color("gray1200", bundle: Bundle.module) 
+    public static let gray1100 = Color("gray1100", bundle: Bundle.module) 
+    public static let gray1000 = Color("gray1000", bundle: Bundle.module) 
+    public static let gray900 = Color("gray900", bundle: Bundle.module) 
+    public static let gray800 = Color("gray800", bundle: Bundle.module) 
+    public static let gray700 = Color("gray700", bundle: Bundle.module) 
+    public static let gray600 = Color("gray600", bundle: Bundle.module) 
+    public static let gray500 = Color("gray500", bundle: Bundle.module) 
+    public static let gray400 = Color("gray400", bundle: Bundle.module) 
+    public static let gray300 = Color("gray300", bundle: Bundle.module) 
+    public static let gray200 = Color("gray200", bundle: Bundle.module) 
+    public static let gray100 = Color("gray100", bundle: Bundle.module) 
+    public static let themeBg = Color("themeBg", bundle: Bundle.module) 
+    public static let bgSubtleSecondaryLevel0 = themeBg
+    public static let bgCanvasDefaultLevel1 = gray300
```

Comment:
These 2 aren't used in the app as they're asymmetric and we still need to figure out how to generate them correctly.